packages/daemon/src/pet-store.js

```diff
@@ -248,7 +238,7 @@ export const makePetStoreMaker = (filePowers, locator) => {
    * @returns {Promise<import('./types.js').PetStore>}
    */
   const makeIdentifiedPetStore = (id, assertValidName) => {
-    if (!validIdPattern.test(id)) {
+    if (!validNumberPattern.test(id)) {
```

Comment:
The `id` param here should have been renamed `formulaNumber` in a previous PR. Likewise in `types.d.ts`.